pkg/controller/association/controller/kb_monitoring.go

```diff
@@ -0,0 +1,66 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package controller
+
+import (
+	kbv1 "github.com/elastic/cloud-on-k8s/pkg/apis/kibana/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+
+	commonv1 "github.com/elastic/cloud-on-k8s/pkg/apis/common/v1"
+	esv1 "github.com/elastic/cloud-on-k8s/pkg/apis/elasticsearch/v1"
+	"github.com/elastic/cloud-on-k8s/pkg/controller/association"
+	"github.com/elastic/cloud-on-k8s/pkg/controller/common/operator"
+	eslabel "github.com/elastic/cloud-on-k8s/pkg/controller/elasticsearch/label"
+	"github.com/elastic/cloud-on-k8s/pkg/controller/elasticsearch/user"
+	"github.com/elastic/cloud-on-k8s/pkg/utils/k8s"
+	"github.com/elastic/cloud-on-k8s/pkg/utils/rbac"
+)
+
+const (
+	// KbAssociationLabelName marks resources created by this controller for easier retrieval.
+	KbAssociationLabelName = "kbassociation.k8s.elastic.co/name"
+	// KbAssociationLabelNamespace marks resources created by this controller for easier retrieval.
+	KbAssociationLabelNamespace = "kbassociation.k8s.elastic.co/namespace"
+	// KbAssociationLabelType marks the type of association.
+	KbAssociationLabelType = "kbassociation.k8s.elastic.co/type"
```

Comment:
I tried to homogenize those comments in all association controllers recently: https://github.com/elastic/cloud-on-k8s/blob/7c39ff040e1e15b3e9b41ceb9779fd9101e938cb/pkg/controller/association/controller/ent_es.go#L24-L32
let's try to stick with the same comment pattern?

Reply:
I will update comments in es_monitoring.go and reuse `KibanaAssociationLabelXX`constants defined in kibana_es.go instead of introducing new ones.